### PR TITLE
Final Voting UI

### PR DESCRIPTION
**Pull request by Raemon**
This prepares for the final voting phase. My goal is to make it so that reviews are much more visible, so final votes can actually be informed by them.

I might want to add some additional features to the ReviewVoteTableRow since it'll be a bit larger and can accommodate more meta-data, but I think that can be a separate PR. (I might want to add Pingback count to the bottom-left of the component, and have the pingbacks appear on hoverover. I think pingbacks are pretty important metadata for the review but they currently aren't that visible).

image

### CODE REVIEW

**Comment by jpaddison3** [comment]
I think "Sort by" needs review is now the wrong label for the sorting of this page.

**Review by jpaddison3 — APPROVED** [review]
ASTLAC

Thanks for making these updates.

File: packages/lesswrong/components/review/ReviewVoteTableRow.tsx [hunk 1]
```diff
@@ -161,7 +181,7 @@ const ReviewVoteTableRow = (
     currentVote: SyntheticReviewVote|null,
   }
 ) => {
```
[the remark this answers is not in the document]
Comment (Raemon) [reply packages/lesswrong/components/review/ReviewVoteTableRow.tsx:183]: Yeah I considered making a whole different component. One of the reasons I hesitated is I think it's possible we want the final version of the TableRow to incorporate the latest changes (i.e. for the entire review, show Review SingleLine Items). But I was still fairly confused about some technical and UI difficulties there.

[the remark this answers is not in the document]
Comment (Raemon) [reply packages/lesswrong/components/review/ReviewVoteTableRow.tsx:183]: I like the idea of having subcomponents that live in one file.

**Pull request merged**